projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx

```diff
@@ -160,9 +176,13 @@ export const findFailingCellInNotebookCells = async (
     });
   } else {
     for (let i = 0; i < cells.length; i++) {
-      i === failingCellIndex
-        ? switchToFailingCell(cells[i])
-        : cells[i].classList.remove('jp-vdk-failing-cell');
+      if (i === failingCellIndex) {
+        switchToFailingCell(cells[i]);
+      } else {
+        cells[i].classList.remove('jp-vdk-failing-cell');
+        const cellPropt = getCellInputAreaPromp(cells[i]);
+        cellPropt?.classList.remove('jp-vdk-failing-cell-prompt')
+      }
```

Comment:
nit: I would consider having the two functions to mark/unmark a cell as failing, then eval those. Think about the interface surface.

Reply:
done